manywheel/README.md

```diff
@@ -1,74 +1,18 @@
-- bump version in setup.py
-- cut branch
-- upload generated docs and add to version selector
+# pytorch/builder/manywheel
 
+## Building docker images
 
-## Building new wheels
+To build all docker images you can use the convenience script:
 
-Run this command:
-
-```
-nvidia-docker run -it --ipc=host --rm -v $(pwd):/remote soumith/manylinux-cuda92:latest bash
-OR
-nvidia-docker run -it --ipc=host --rm -v $(pwd):/remote soumith/manylinux-cuda100:latest bash
-```
-
-Exit the container, then back in the host, run:
-
-```
-# versioned, build all
-export PYTORCH_REPO=pytorch
-export PYTORCH_BRANCH=v1.1.0
-export PYTORCH_BUILD_VERSION=1.1.0
-export PYTORCH_BUILD_NUMBER=1
-export TORCH_CONDA_BUILD_FOLDER=pytorch-1.1.0
-export TORCH_PACKAGE_NAME=torch
-export PIP_UPLOAD_FOLDER=""
-export NIGHTLIES_ROOT_FOLDER="/private/home/soumith/local/builder/binaries_v1.1.0"
-cd ../cron
-./build_multiple.sh manywheel all all
-./remote/build.sh
-
-# single nightly build
-export PYTORCH_REPO=pytorch
-export PYTORCH_BRANCH=master
-export PYTORCH_BUILD_VERSION=1.2.5
-export PYTORCH_BUILD_NUMBER=1
-export TORCH_CONDA_BUILD_FOLDER=pytorch-nightly
-export TORCH_PACKAGE_NAME=torch
-export PIP_UPLOAD_FOLDER=""
-export NIGHTLIES_ROOT_FOLDER="/private/home/soumith/local/builder/binaries_nightly"
-cd ../cron
-./build_multiple.sh manywheel 3.6m cu100
-./remote/build.sh
-
-```
-
-Once done, upload wheels via:
-
-```
-./upload.sh
+```bash
+# Build without pushing
+manywheel/build_all_docker.sh
+# Build with pushing
```

Comment:
Is this comment supposed to have another command come after it?

Reply:
Should be resolved now